### DISCUSSION TITLE
Node.js `installGlobals()` breaks Jest line numbers / source maps

### DISCUSSION THREAD

**Discussion by esamattis** [remix-run/remix#2262] [General]
This is not necessarily an issue in Remix but posting this here since there doesn't seem to be any other reports about this.

It seems that if you use 

```ts
import { installGlobals } from "@remix-run/node";
installGlobals();
```

to add global Fetch and Request / Response classes in Jest line numbering in expectation / assertion messages will be off. 

No sure why this happens because `installGlobals()` doesn't seem to do anything related to it:

https://github.com/remix-run/remix/blob/5b8a0ce0aa0201aa2402fc41405ffbe89605963b/packages/remix-node/globals.ts#L44-L59

For my use case I was able to just import the Request class from `node-fetch`.

**Comment by petergledhillinclusive** [comment]
Thanks for pointing this out,  I have experienced this too and it makes our jest tests very hard to debug.  We seem to get the correct name of the failing test but the line numbers are from a different test.

I tried using node-fetch but it doesn't cover the full range of changes that remix 'installGlobals' does.
